cvat-ui/src/components/task-page/task-page.tsx

```diff
@@ -83,6 +84,7 @@ class TaskPageComponent extends React.PureComponent<Props> {
                     </Col>
                 </Row>
                 <ModelRunnerModal />
+                <MoveTaskModal />
```

Comment:
I would vote for more meaningful name.. MoveTaskToProjectModal?

Reply:
In future feature implementation it can be moving outside of the project